src/util/file_util.cpp

```diff
@@ -107,7 +107,13 @@ void delete_directory(const std::string &path)
     struct dirent *ent;
 
     while((ent=readdir(dir))!=NULL)
```

Comment:
This is not my code. I fear that if I change this it will expand the context area and you'll then find something else above that you want me to change too.